shamuna/van.css

```diff
@@ -0,0 +1,67 @@
+body{
+    background-color:#DACC98;
+    margin:0;
+ 
+  
+}
+
+.header{
+    width: 100%;
+    height: 48px;
+    
+   
+  
+    text-align: left;
+    padding-top: 47px;
+}
+
+a{
+    display: inline;
+    text-decoration: none;
+    margin-right: 90px;
+}
+.container{
+    width:300%;
+    height: 418px;
+    padding-top: 176px;
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    overflow-x: scroll;
+   
+}
+.container::-webkit-scrollbar
+{
+    width:0px;
+}
+
+.small-container{
```

Comment:
We don't need 2 containers here, let's remove one of them and use container instead